MegaQuests/src/main/java/com/elmer/megaquests/listeners/questlisteners/BlockBreakQuestsListener.java

```diff
@@ -13,24 +13,29 @@
 
 public class BlockBreakQuestsListener implements Listener {
     MegaQuests megaQuests;
-    public BlockBreakQuestsListener(MegaQuests megaQuests){
+
+    public BlockBreakQuestsListener(MegaQuests megaQuests) {
         this.megaQuests = megaQuests;
     }
 
     @EventHandler
-    public void onBlockBreak (BlockBreakEvent e){
-        Player player = e.getPlayer();
-        Block brokenBlock = e.getBlock();
+    public void onBlockBreak(BlockBreakEvent event) {
+        Player player = event.getPlayer();
+        Block brokenBlock = event.getBlock();
         Inventory questGUI = megaQuests.getQuestGUICommand().getQuestGUI();
         QuestManager questManager = megaQuests.getQuestManager();
 
-
-        for (int i = questManager.getStartingSlot(); i < questManager.getEndingSlot(); i++){
-            if (brokenBlock.getType().equals(Material.DIAMOND_ORE) && questGUI.getItem(i).getType().equals(Material.DIAMOND_ORE) && Quests.MINEDIAMONDORE.isEnabled() && !Quests.MINEDIAMONDORE.isCompleted()){
-                questManager.checkCompletion(player, Quests.MINEDIAMONDORE,1);
-            }
-            if (brokenBlock.getType().equals(Material.IRON_ORE) && questGUI.getItem(i).getType().equals(Material.IRON_ORE) && Quests.MINEIRONORE.isEnabled() && !Quests.MINEIRONORE.isCompleted()){
-                questManager.checkCompletion(player, Quests.MINEIRONORE, 1);
+        for (int i = questManager.getStartingSlot(); i < questManager.getEndingSlot(); i++) {
+            if (brokenBlock.getType().equals(Material.DIAMOND_ORE)
+                    && questGUI.getItem(i).getType().equals(Material.DIAMOND_ORE)
+                    && Quests.MINE_DIAMOND_ORE.isEnabled()
+                    && !Quests.MINE_DIAMOND_ORE.isCompleted()) {
+                questManager.checkCompletion(player, Quests.MINE_DIAMOND_ORE, 1);
+            } else if (brokenBlock.getType().equals(Material.IRON_ORE)
+                    && questGUI.getItem(i).getType().equals(Material.IRON_ORE)
+                    && Quests.MINE_IRON_ORE.isEnabled()
+                    && !Quests.MINE_IRON_ORE.isCompleted()) {
+                questManager.checkCompletion(player, Quests.MINE_IRON_ORE, 1);

```

Comment:
That's too bad. The design is fundamentally flawed. 
I won't be able to fix it, try doing it yourself. 
The problem is that if you want to scale your quests, the number of conditions will grow proportionally, making it difficult to fix bugs, add new code and generally maintain that code. 

Also, if you have multiple checks in a condition, your condition will be very bloated in length, it's better to put each new condition on a new line as it's easier to read.